app/views/contestants/index.html.erb

```diff
@@ -0,0 +1,10 @@
+<h1>All Contestants</h1>
+
+<% @contestants.each do |contestant| %>
+      <h3><%= contestant.name %></h3>
+      <% project_names = [] %>
+      <% contestant.projects.each do |project| %>
+        <% project_names << project.name %>
+        <%end%>
+        <p>Projects: <%= "#{project_names.join(", ")}" %>
+<% end %>
```

Comment:
I think there could have been a shorter way to accomplish what you have here using `map`.

```erb
<% @contestants.each do |contestant| %>
  <h3><%= contestant.name %></h3>

  <p>Projects: <%= contestant.projects.map do |project| %>
                 <% project.name %>
               <% end.join(", ") %>
  </p>
<% end %>
```

This is just a suggestion to make your code cleaner and easier to read. What you have also work too.